person_capture/gui_app.py

```
@@ -10036,6 +10036,16 @@ def _side_gain(src_dev: np.ndarray, dst_dev: np.ndarray, side: int) -> Optional[
             except Exception:
                 lowfreq = 0.0
 
+            try:
+                black_chroma_neutralize = self._float_env(
+                    "PC_WIC_YUV444_COLOR_MATCH_BLACK_CHROMA_NEUTRALIZE",
+                    0.75,
+                    min_value=0.0,
+                    max_value=1.0,
+                )
+            except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)